src/backend/utils/indexer.js

```diff
@@ -67,6 +67,16 @@ const createFieldsFromFilter = (filter) => {
   }
 };
 
+const sortCase = (filter) => {
```

Comment:
Can we rename this for consistency to `sortFromFilter`?

Reply:
of course